drivers/sdmmc/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_SDMMC
+    bool "SDIO/SD/MMC interface"
+    depends on HAS_PERIPH_SDMMC
+    select MODULE_PERIPH_SDMMC
+    select MODULE_ZTIMER_MSEC if !ZTIMER_USEC
```

Comment:
```suggestion
    select MODULE_ZTIMER_MSEC if !MODULE_ZTIMER_USEC
```

Reply:
I'm not sure, in case of `ztimer` there is a symbol `ZTIMER_USEC` that is used to enable `ztimer_usec` which then selects `MODULE_ZTIMER_USEC`.

Reply:
Yes, I had trouble with dependency loop when using `MODULE_ZTIMER_USEC` caused in `sys/ztimer/Kconfig`
```
Dependency loop
===============

MODULE_ZTIMER_MSEC (defined at /home/gs/src/RIOT-Xtensa-ESP.esp-idf-4.4/sys/ztimer/Kconfig:67), with definition...

config MODULE_ZTIMER_MSEC
	bool "Milliseconds"
	select MODULE_ZTIMER
```